### ISSUE TITLE
Manual test run on Windows x64 for 0.22.x Release 3 (Beta channel)

### ISSUE THREAD

**Issue by srirambv** [brave/browser-laptop#14004]
## Per release specialty tests
 - [x] Cannot set property 'enabled' of null when launching or closing Brave.. ([#13991](https://github.com/brave/browser-laptop/issues/13991))
 - [x] Extension popups crash browser. ([#13983](https://github.com/brave/browser-laptop/issues/13983))
 - [x] Display error when ledger server is not working. ([#13972](https://github.com/brave/browser-laptop/issues/13972))
 - [x] Find results stay highlighted even after you hit escape. ([#13931](https://github.com/brave/browser-laptop/issues/13931))
 - [ ] URL Heart (publisher toggle) button should be able to include sites outside of minimum requirements. ([#13879](https://github.com/brave/browser-laptop/issues/13879))
 - [x] Tab's first attach is not composited. ([#13827](https://github.com/brave/browser-laptop/issues/13827))
 - [ ] On macOS, switched-to tab is blank until window resize / move. ([#13825](https://github.com/brave/browser-laptop/issues/13825))
 - [x] Drag and drop of a tab shows tab content as white. ([#13815](https://github.com/brave/browser-laptop/issues/13815))
 - [x] Move webrtc options from advanced to security pref panel. ([#13805](https://github.com/brave/browser-laptop/issues/13805))
 - [x] Infinite loading spinner when detaching a loading tab (sometimes). ([#13803](https://github.com/brave/browser-laptop/issues/13803))
 - [ ] Tab is destroyed if switched during first attach. ([#13798](https://github.com/brave/browser-laptop/issues/13798))
 - [x] Some Top site tiles are missing favicons. ([#13728](https://github.com/brave/browser-laptop/issues/13728))
 - [x] Sort ledger table based on %. ([#13721](https://github.com/brave/browser-laptop/issues/13721))
 - [x] Re-pin of tab can take multiple tries for tab to be pinned again. ([#13708](https://github.com/brave/browser-laptop/issues/13708))
 - [ ] Tabs disappear when combining windows (magically disappearing tabs). ([#13706](https://github.com/brave/browser-laptop/issues/13706))
 - [ ] Sometimes attach/detach of tab causes crash. ([#13698](https://github.com/brave/browser-laptop/issues/13698))
 - [ ] [FEATURE REQUEST] Add an option to disable WebRTC in about:preferences. ([#13668](https://github.com/brave/browser-laptop/issues/13668))
 - [ ] Recover your Brave wallet dialog shown after successful recovery. ([#13645](https://github.com/brave/browser-laptop/issues/13645))
 - [x] Ctrl+f search bar not functional. ([#13602](https://github.com/brave/browser-laptop/issues/13602))
 - [ ] Remind the user to generate a backup file. ([#13425](https://github.com/brave/browser-laptop/issues/13425))
 - [ ] Reconcile shouldn't happen when ledger table is empty . ([#13325](https://github.com/brave/browser-laptop/issues/13325))
 - [x] Bookmark search does not work, cannot type in letters in the search field.. ([#13322](https://github.com/brave/browser-laptop/issues/13322))
 - [x] Running from source (npm run start) often crashes on Windows. ([#13224](https://github.com/brave/browser-laptop/issues/13224))
 - [x] Seeing youtube#channel:{channelId} in the about:preferences#payments panel. ([#13217](https://github.com/brave/browser-laptop/issues/13217))
 - [ ] Ledger doesn't count visits to or time spent reading PDFs. ([#12792](https://github.com/brave/browser-laptop/issues/12792))
 - [x] URL Heart (include for payment) button disabled even after time requirements satisfied. ([#12675](https://github.com/brave/browser-laptop/issues/12675))
 - [ ] Formatting for Saved Site Exceptions (deleted YT channels) should match ledger table. ([#12271](https://github.com/brave/browser-laptop/issues/12271))
 - [ ] Textbox not large enough to display large balances. ([#12220](https://github.com/brave/browser-laptop/issues/12220))
 - [ ] Loading/progress indicator for wallet recovery. ([#11790](https://github.com/brave/browser-laptop/issues/11790))
 - [ ] With many tabs, startup is very slow. ([#11424](https://github.com/brave/browser-laptop/issues/11424))
 - [x] Green tick on payments icon disappears by moving to sub level of the verified site. ([#10484](https://github.com/brave/browser-laptop/issues/10484))
 - [ ] Add a new option to right-click for history and bookmarks. ([#6547](https://github.com/brave/browser-laptop/issues/6547))

## Installer

- [ ] Check that installer is close to the size of last release.
- [ ] Check signature: If OS Run `spctl --assess --verbose /Applications/Brave.app/` and make sure it returns `accepted`.  If Windows right click on the installer exe and go to Properties, go to the Digital Signatures tab and double click on the signature.  Make sure it says "The digital signature is OK" in the popup window.
- [ ] Check Brave, muon, and libchromiumcontent version in `about:brave` and make sure it is EXACTLY as expected.

## Last changeset test

- [ ] Test what is covered by the last changeset (you can find this by clicking on the SHA in about:brave)

## Data

- [ ] Make sure that data from the last version appears in the new version OK
- [ ] With data from the last version, test that
  - [ ] cookies are preserved
  - [ ] pinned tabs can be opened
  - [ ] pinned tabs can be unpinned
  - [ ] unpinned tabs can be re-pinned
  - [ ] opened tabs can be reloaded
  - [ ] bookmarks on the bookmark toolbar can be opened
  - [ ] bookmarks in the bookmark folder toolbar can be opened

## About pages

- [x] Test that about:adblock loads
- [x] Test that about:autofill loads
- [x] Test that about:bookmarks loads bookmarks
- [x] Test that about:downloads loads downloads
- [x] Test that about:extensions loads
- [x] Test that about:history loads history
- [x] Test that about:passwords loads
- [x] Test that about:styles loads
- [x] Test that about:welcome loads
- [x] Test that about:preferences changing a preference takes effect right away
- [x] Test that about:preferences language change takes effect on re-start

## Bookmarks

- [ ] Test that creating a bookmark on the bookmarks toolbar with the star button works
- [ ] Test that creating a bookmark on the bookmarks toolbar by dragging the un/lock icon works
- [ ] Test that creating a bookmark folder on the bookmarks toolbar works
- [ ] Test that moving a bookmark into a folder by drag and drop on the bookmarks folder works
- [ ] Test that clicking a bookmark in the toolbar loads the bookmark.
- [ ] Test that clicking a bookmark in a bookmark toolbar folder loads the bookmark.
- [ ] Test that a bookmark on the bookmark toolbar can be removed via context menu
- [ ] Test that a bookmark in a bookmark folder on the bookmark toolbar can be removed via context menu
- [ ] Test that a bookmark subfolder can be removed via context menu
- [ ] Test that a bookmark folder on the bookmark toolbar can be removed via context menu

## Context menus

- [ ] Make sure context menu items in the URL bar work
- [ ] Make sure context menu items on content work with no selected text
- [ ] Make sure context menu items on content work with selected text
- [ ] Make sure context menu items on content work inside an editable control on `about:styles` (input, textarea, or contenteditable)

## Find on page

- [ ] Ensure search box is shown with shortcut
- [ ] Test successful find
- [ ] Test forward and backward find navigation
- [ ] Test failed find shows 0 results
- [ ] Test match case find

## Keyboard Shortcuts

- [x] Open a new window: `Command` + `N` (macOS) || `Ctrl` + `N` (Win/Linux)
- [x] Open a new tab: `Command` + `T` (macOS) || `Ctrl` + `T` (Win/Linux)
- [x] Open a new private tab: `Command` + `Shift` + `P` (macOS) || `Ctrl` + `Shift` + `P` (Win/Linux)
- [x] Reopen the latest closed tab: `Command` + `Shift` + `t` (macOS) || `Ctrl` + `Shift` + `t` (Win/Linux)
- [ ] Reopen the latest closed window: `Command` + `Shift` + `Option` + `T` (macOS) || `Ctrl` + `Shift` + `Alt` + `T` (Win/Linux)
- [x] Jump to the next tab: `Command` + `Option` + `->` (macOS) || `Ctrl` + `PgDn` (Win/Linux)
- [x] Jump to the previous tab: `Command` + `Option` + `<-` (macOS) || `Ctrl` + `PgUp` (Win/Linux)
- [x] Jump to the next tab: `Ctrl` + `Tab` (macOS/Win/Linux)
- [x] Jump to the previous tab: `Ctrl` + `Shift` + `Tab` (macOS/Win/Linux)
- [x] Open Brave preferences: `Command` + `,` (macOS) || `Ctrl` + `,` (Win/Linux)
- [x] Jump into the URL bar: `Command` + `L` (macOS) || `Ctrl` + `L` (Win/Linux)
- [x] Reload page: `Command` + `R` (macOS) || `Ctrl` + `R` (Win/Linux)
- [x] Select All: `Command` + `A` (macOS) || `Ctrl` + `A` (Win/Linux)
- [x] Copying text: `Command` + `C` (macOS) || `Ctrl` + `C` (Win/Linux)
- [x] Pasting text: `Command` + `V` (macOS) || `Ctrl` + `V` (Win/Linux)
- [x] Minimize Brave: `Command` + `M` (macOS) || `Ctrl` + `M` (Win/Linux)
- [x] Quit Brave: `Command` + `Q` (macOS) || `Ctrl` + `Q` (Win/Linux)

## Tabs, Pinning and Tear off tabs

- [x] Test that tabs are pinnable
- [x] Test that tabs are unpinnable
- [ ] Test that tabs are draggable to same tabset
- [ ] Test that tabs are draggable to alternate tabset
- [ ] Test that tabs can be teared off into a new window
- [ ] Test that you are able to reattach a tab that is teared off into a new window
- [ ] Test that tab pages can be closed
- [ ] Test that tab pages can be muted
- [ ] Test that tabs can be cloned

## Downloads

- [x] Test downloading a file works and that all actions on the download item works

## Fullscreen

- [x] Test that entering full screen window works View -> Toggle Full Screen. And exit back (Not Esc)
- [x] Test that entering HTML5 full screen works. And Esc to go back. (youtube.com)

## Zoom

- [x] Test zoom in / out shortcut works
- [x] Test hamburger menu zooms
- [x] Test zoom saved when you close the browser and restore on a single site
- [x] Test zoom saved when you navigate within a single origin site
- [x] Test that navigating to a different origin resets the zoom

## Printing

- [x] Test that you can print a PDF

## Extensions/Plugins tests

- [ ] Enable each extension one by one under `about:preferences#extensions` and ensure that the browser doesn't become unresponsive

### Widevine
  - [x] Test that you can log into Netflix and start a show

### Flash tests
  - [ ] Test that flash placeholder appears on http://www.homestarrunner.com
  - [ ] Test with flash enabled in preferences, auto play option is shown when visiting http://www.homestarrunner.com

## Autofill tests

- [ ] Test that autofill works on http://www.roboform.com/filling-test-all-fields
- [ ] Verify clicking `Next` button on https://www.paypal.com/signin doesn't crash the browser

## Geolocation

- [x] Check that https://developer.mozilla.org/en-US/docs/Web/API/Geolocation/Using_geolocation is blocked due to cross-origin iframes
- [x] Check that https://browserleaks.com/geo works and shows correct location
- [x] Check that https://html5demos.com/geo/ works but doesn't require an accurate location

## Crash Reporting

- [x] Check that loading `chrome://crash` causes the new tab to crash 
- [x] Check that `chrome://crashes` lists all the crashes and includes both Crash Report ID & Local Crash ID
- [ ] Verify the crash ID matches the report on brave stats

## Performance test

_Each start should take less than 7 seconds_
- [ ] Enable only sync (new sync group).
- [ ] Enable only sync with a large sync group (many entries).
- [ ] Enable only payments.
- [ ] Only import a large set of bookmarks.
- [ ] Combine sync, payments, and a large set of bookmarks.

## Bravery settings

- [x] Check that HTTPS Everywhere works by loading http://https-everywhere.badssl.com/
- [x] Turning HTTPS Everywhere off and shields off both disable the redirect to https://https-everywhere.badssl.com/
- [x] Check that ad replacement works on http://slashdot.org
- [x] Check that toggling to blocking and allow ads works as expected
- [x] Test that clicking through a cert error in https://badssl.com/ works
- [x] Test that Safe Browsing works (https://www.raisegame.com/)
- [x] Turning Safe Browsing off and shields off both disable safe browsing for https://www.raisegame.com/
- [x] Visit https://brianbondy.com/ and then turn on script blocking, nothing should load. Allow it from the script blocking UI in the URL bar and it should work
- [x] Test that about:preferences default Bravery settings take effect on pages with no site settings
- [x] Test that 3rd party storage results are blank at https://jsfiddle.net/7ke9r14a/9/ when 3rd party cookies are blocked and not blank when 3rd party cookies are unblocked
### Fingerprint Tests
  - [x] Visit https://jsfiddle.net/bkf50r8v/13/, ensure 3 blocked items are listed in shields. Result window should show `got canvas fingerprint 0`  and  `got webgl fingerprint 00`
  - [x] Test that audio fingerprint is blocked at https://audiofingerprint.openwpm.com/ only when `Block all fingerprinting protection` is on
  - [x] Test that Brave browser isn't detected on https://extensions.inrialpes.fr/brave/
  - [x] Test that https://diafygi.github.io/webrtc-ips/ doesn't leak IP address when `Block all fingerprinting protection` is on

## Content tests

- [x] Go to https://brianbondy.com/ and click on the twitter icon on the top right. Test that context menus work in the new twitter tab
- [x] Load twitter and click on a tweet so the popup div shows.   Click to dismiss and repeat with another div. Make sure it shows
- [x] Go to https://www.bennish.net/web-notifications.html and test that clicking on 'Show' pops up a notification asking for permission. Make sure that clicking 'Deny' leads to no notifications being shown
- [x] Go to https://trac.torproject.org/projects/tor/login and make sure that the password can be saved. Make sure the saved password shows up in `about:passwords`. Then reload https://trac.torproject.org/projects/tor/login and make sure the saved credentials aren't autofilled instead shows saved id as a dropdown under the login field
- [x] Open `about:styles` and type some misspellings on a textbox, make sure they are underlined.
- [x] Make sure that right clicking on a word with suggestions gives a suggestion and that clicking on the suggestion replaces the text
- [x] Make sure that Command + Click (Control + Click on Windows, Control + Click on Ubuntu) on a link opens a new tab but does NOT switch to it.  Click on it and make sure it is already loaded
- [x] Open an email on http://mail.google.com/ or inbox.google.com and click on a link. Make sure it works
- [x] Test that PDF is loaded over https at https://basicattentiontoken.org/BasicAttentionTokenWhitePaper-4.pdf
- [x] Test that PDF is loaded over http at http://www.pdf995.com/samples/pdf.pdf
- [x] Test that https://mixed-script.badssl.com/ shows up as grey not red (no mixed content scripts are run).
- [x] Test that hovering the cursor over a link changes the cursor into a pointer (hand)
- [x] Visit https://dlptest.com/ftp-test/ and ensure that basic authentication is working by clicking on ftp://ftp.dlptest.com/ and ensuring you're prompted for credentials
- [x] Test that WebSockets are working by ensuring http://slither.io/ runs once "Play" has been clicked.

## Ledger

- [x] Verify wallet is auto created after enabling payments
- [x] Verify monthly budget and account balance shows correct BAT and USD value
- [x] Click on `add funds` and click on each currency and verify it shows wallet address and QR Code
- [x] Verify that Brave BAT wallet address can be copied
- [x] Verify adding funds via any of the currencies flows into BAT Wallet after specified amount of time
- [ ] Verify adding funds to an existing wallet with amount, adjusts the BAT value appropriately
- [x] Change min visit and min time in advance setting and verify if the publisher list gets updated based on new setting
- [x] Visit nytimes.com for a few seconds and make sure it shows up in the Payments table
- [x] Check that disabling payments and enabling them again does not lose state
- [x] Upgrade from older version
  - [x] Verify the wallet isn't corrupted upon upgrade (balance is retained and wallet backup code isn't corrupted)
  - [x] Verify publishers list is not lost after upgrade
### Ledger Media (To be verified on YouTube and Twitch) 
  - [ ] Visit any YouTube/Twitch video in a normal/session tab and ensure the video publisher name is listed in ledger table
  - [ ] Visit any YouTube/Twitch video in a private tab and ensure the video publisher name is not listed in ledger table
  - [ ] Visit any live YouTube/Twitch video and ensure the time spent is shown under ledger table
  - [ ] Visit any embeded YouTube/Twitch video and ensure the video publisher name is listed in ledger table
  - [ ] Ensure total time spent is correctly calculated for each publisher video
  - [ ] Ensure total time spent is correctly calculated when switching to YouTube/Twitch video from an embeded video
  - [ ] Ensure YouTube/Twitch publishers are not listed when `Allow contributions to video` is disabled in adavanced settings
  - [ ] Ensure existing YouTube/Twitch publishers are not lost when `Allow contributions to video` is disabled in adavanced settings
  - [ ] Ensure YouTube/Twitch publishers is listed but not included when `auto-include` is disabled
  - [ ] Update Advanced settings to different time/visit value and ensure YouTube/Twitch videos are added to ledger table once criteria is met
  - [ ] Perform a contribution while YouTube/Twitch channels are included on the ledger. Ensure the channels are listed on the contribution statement
  - [ ] Verify that you are able to delete YouTube/Twitch publishers from ledger table
  - [ ] Verify that you are able to re-add YouTube/Twitch publishers to ledger table
  - [ ] Verify if you minimize a Twitch video (Stream/VOD) and navigate around the site, the video is counted in ledger

## Session storage

Do not forget to make a backup of your entire `~/Library/Application\ Support/Brave` folder.

- [x] Temporarily move away your `~/Library/Application\ Support/Brave/session-store-1` and test that clean session storage works. (`%appdata%\Brave` in Windows, `./config/brave` in Ubuntu)
- [x] Test that windows and tabs restore when closed, including active tab.
- [x] Move away your entire `~/Library/Application\ Support/Brave` folder (`%appdata%\Brave` in Windows, `./config/brave` in Ubuntu)

## Cookie and Cache

- [x] Make a backup of your profile, turn on all clearing in preferences and shut down.  Make sure when you bring the browser back up everything is gone that is specified.
- [x] Go to http://samy.pl/evercookie/ and set an evercookie. Check that going to prefs, clearing site data and cache, and going back to the Evercookie site does not remember the old evercookie value.

## Update tests

- [x] Test that updating using `BRAVE_UPDATE_VERSION=0.8.3` env variable works correctly.
- [x] Test that using `BRAVE_ENABLE_PREVIEW_UPDATES=TRUE` env variable works and prompts for preview build updates.

**Comment by srirambv** [comment]
Closing in favour of #14053
